templates/pytorch/nightly/cpp-operations.libsonnet

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local base = import "base.libsonnet";
+local timeouts = import "../../timeouts.libsonnet";
+local tpus = import "../../tpus.libsonnet";
+
+{
+  local operations = base.PyTorchTest {
+    modelName: "cpp-operations",
+    command: [
+      "bash",
+      "pytorch/xla/test/cpp/run_tests.sh",
+    ],
+    regressionTestConfig: {}
```

Comment:
Would it be clearer to use `null` here instead of `{}`? It would make JSonnet throw an error if you were to accidentally try to override a field.

Reply:
Sure, sounds good. Switched to that and re-pushed the yamls and re-ran them to make sure the change works